at-sample/dubbo-samples-seata/dubbo-samples-seata-account/src/main/resources/application.yml

```diff
@@ -18,7 +18,7 @@ spring:
         name: AccountApplication
     datasource:
         driver-class-name: com.mysql.cj.jdbc.Driver
-        url: jdbc:mysql://${mysql.address:127.0.0.1}:${mysql.port:3306}/seata?serverTimezone=Asia/Shanghai&useSSL=false&useUnicode=true&characterEncoding=utf8&zeroDateTimeBehavior=convertToNull&useOldAliasMetadataBehavior=true
+        url: jdbc:mysql://${mysql.address:127.0.0.1}:${mysql.port:3306}/seata?serverTimezone=Asia/Shanghai&allowPublicKeyRetrieval=true&useSSL=false&useUnicode=true&characterEncoding=utf8&zeroDateTimeBehavior=convertToNull&useOldAliasMetadataBehavior=true
```

Comment:
This combination doesn't seem to make sense; allowPublicKeyRetrieval and useSSL are generally enabled simultaneously. Q: I would like to know whether the password authentication related configuration is enabled in your database, and what the connection string of the final database (IP information can be removed) is like?